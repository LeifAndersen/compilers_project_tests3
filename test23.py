def a():
 def a():
  return 3
 return a() + 3
a()

