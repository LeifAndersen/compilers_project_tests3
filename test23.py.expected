(program
 (define a (void))
 (set-global!
  a
  (lambda ()
    (call/ec
     (lambda (return)
       (let ((a (void)))
         (set! a (lambda () (call/ec (lambda (return) (let () (return 3))))))
         (return (+ (a) 3)))))))
 ((get-global a)))

